test/DxfParser.test.js

```diff
@@ -65,10 +65,28 @@ describe('Parser', function() {
 		tables.viewPort.should.eql(JSON.parse(expected));
 	});
 
-	it('should parse a complex BLOCKS section', function() {
-		verifyDxf(path.join(__dirname, 'data', 'blocks.dxf'))
-	});
-	
+    it('should parse HATCH', function() {
+		verifyDxf
```

Comment:
The new way to do this would be to just call `verifyDxf(path.join(__dirname, 'data', 'hatch.dxf'));` and the result will be suffixed with `.received.json` instead of `.actual.json`.